judge/handler.py

```diff
@@ -106,29 +116,31 @@ def process_problem(code: str, contest: int, name: str, statement: str, input_fo
             code=code, contest=c, name=name, statement=statement,
             input_format=input_format, output_format=output_format,
             difficulty=difficulty, time_limit=time_limit, memory_limit=memory_limit,
-            file_format=file_format, start_code=start_code, max_score=max_score,
+            file_format=file_format, start_code=starting_code, max_score=max_score,
             compilation_script=compilation_script,
-            test_script=test_script, setter_solution=setter_solution)
+            test_script=test_script)
 
         if not os.path.exists(os.path.join('content', 'problems', p.code)):
             # Create the problem directory explictly if not yet created
             # This will happen when both compilation_script and test_script were None
             os.makedirs(os.path.join('content', 'problems', p.code))
 
-        if cp_comp_script:
+        if compilation_script is None:
             # Copy the default comp_script if the user did not upload custom
-            run(['cp', os.path.join('judge', compilation_script),
-                 os.path.join('content', 'problems', p.code, 'compilation_script.sh')])
+            copyfile(os.path.join('judge', 'default', 'compilation_script.sh'),
+                     os.path.join('content', 'problems', p.code, 'compilation_script.sh'))
             p.compilation_script = os.path.join('content', 'problems',
```

Comment:
@vbsinha check this change. I think what you say has been unchanged.

Reply:
I forgot what create did. My memory has taken a toss. My bad.